src/webui/www/private/scripts/mocha-init.js

```diff
@@ -93,6 +93,34 @@ let copyCommentFN = function() {};
 let setQueuePositionFN = function() {};
 let exportTorrentFN = function() {};
 
+const baseModalOptions = Object.freeze({
```

Comment:
@Chocobo1, I'm not sure if this is the best way to do it. I think I should keep callbacks out of it for now.